lightkurve/interact.py

```diff
@@ -517,6 +534,8 @@ def save_lightcurve():
                 selected_indices = np.array(tpf_source.selected.indices)
                 selected_mask = np.isin(pixel_index_array, selected_indices)
                 lc_new = tpf.to_lightcurve(aperture_mask=selected_mask)
+                if transform_func is not None:
+                    lc_new = transform_func(lc_new)
```

Comment:
It looks like lines 490-494, 534-538, and perhaps 438-440 do the same thing.  Would it be good to separate this into a single helper function (`_create_lc` or whatever) so we reduce the likelihood of bugs in future updates?